src/shared/treeview/nodes/awsTreeNodeBase.ts

```diff
@@ -13,4 +14,8 @@ export abstract class AWSTreeNodeBase extends TreeItem {
     public getChildren(): Thenable<AWSTreeNodeBase[]> {
         return Promise.resolve([])
     }
+
+    public refresh(): void {
+        ext.awsExplorer.refresh(this)
```

Comment:
Direct refresh method on nodes makes more sense than having a command dedicated to the task.

Reply:
Is this the only reason to expose `awsExplorer`? Worth considering exposing it in a different way so we can access nodes and node contents from elsewhere. It brings up an entire massive caching discussion though...

Reply:
The only reason for this PR, though I think we do want to expose the tree to other parts of the code. One other reason for avoiding the command is that the Cloud9 mangles nodes passed via commands.